src/context/apiClient.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import * as rest from "../rest/interface";
+import { ClientConfig, IsValidConfig } from "../models/client_config";
+
+const { createContext, useState } = React;
+
+const configPath = "/config.json";
+
+interface ApiClientContextType {
+  apiClient: rest.Evergreen;
+  actions: {
+    tryLoadConfig: () => void;
```

Comment:
This isn't called from anywhere as far as I can tell, but it should be one of the first things that happens

Reply:
good point--will update

Reply:
updated that so that `tryLoadConfig` is called when the `ApiClientProvider` mounts. That way all consumers of the api client context will be updated when apiClient is ready.